packages/ui/src/components/Modal/style.less

```diff
@@ -0,0 +1,97 @@
+@import "antd/lib/modal/style/index.less";
+@import (reference) "@open-condo/ui/src/tokens/variables.less";
+
+@condo-modal-margin-vertical: @condo-global-spacing-40;
+@condo-modal-margin-horizontal: @condo-global-spacing-40;
+@condo-modal-border-width: @condo-global-border-width-default;
+@condo-modal-padding: @condo-global-spacing-40;
+@condo-modal-half-padding: (@condo-modal-padding / 2);
+@condo-modal-half-padding-without-border: @condo-modal-half-padding - @condo-modal-border-width;
+@condo-modal-close-icon-width: 24px;
+@condo-modal-close-icon-height: 24px;
+@condo-modal-close-icon-top: @condo-modal-padding +
+  ((@condo-global-line-height-h3 - @condo-modal-close-icon-height) / 2);
+@condo-modal-header-padding-right: (@condo-modal-padding * 2) + @condo-modal-close-icon-width;
+@condo-modal-header-scrolling-border-bottom-color: @condo-global-color-gray-3;
+@condo-modal-footer-buttons-gap: @condo-global-spacing-20;
+
+.condo-modal-root {
+  .condo-modal-mask {
+    background-color: @condo-global-color-gray-7;
+    opacity: 0.5;
+  }
+
+  .condo-modal {
+    max-width: calc(100vw - @condo-modal-margin-horizontal * 2);
+    margin: @condo-modal-margin-vertical 0;
+
+    .condo-modal-header {
+      flex-shrink: 0;
+      padding:
+        @condo-modal-padding
+        @condo-modal-header-padding-right
+        @condo-modal-half-padding-without-border
+        @condo-modal-padding;
+      border-bottom: @condo-global-border-width-default solid transparent;
+      transition: border-bottom-color @condo-global-transition-duration-default;
+    }
+
+    &.condo-modal-scrolling {
+      .condo-modal-header {
+        border-bottom-color: @condo-modal-header-scrolling-border-bottom-color;
+      }
+    }
+
+    .condo-modal-close {
+      top: @condo-modal-close-icon-top;
+      right: @condo-modal-padding;
+
+      .condo-modal-close-x {
+        display: flex;
+        align-items: center;
+        justify-content: center;
+        width: @condo-modal-close-icon-width;
+        height: @condo-modal-close-icon-height;
+        line-height: @condo-modal-close-icon-height;
+
+        div {
+          width: @condo-modal-close-icon-width;
+          height: @condo-modal-close-icon-height;
+        }
+      }
+    }
+
+    .condo-modal-content {
+      display: flex;
+      flex-direction: column;
+      max-height: calc(100vh - (@condo-modal-margin-vertical * 2));
+      padding: 0;
+      overflow: hidden;
+      border-radius: @condo-global-border-radius-large;
+    }
+
+    .condo-modal-body {
+      padding: @condo-modal-half-padding @condo-modal-padding @condo-modal-padding;
+      overflow: auto;
+      overflow: overlay; // NOTE scrollbar does not reduce content width. Not supported in all browsers
+
+      .condo-modal-wrapper {
+        width: fit-content;
+      }
+    }
+
+    .condo-modal-footer {
+      display: flex;
+      flex-shrink: 0;
+      flex-wrap: wrap-reverse;
+      gap: @condo-modal-footer-buttons-gap;
```

Comment:
Gap is relatively new css property. ([Safari for example started supporting it only in 2021](https://caniuse.com/flexbox-gap)). So may be use margin with `:not(:last-child)`

Reply:
Removed this css property. I left this implementation to the discretion of the developers who will add footers